packages/swhid/swhid.0.1/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "OCaml library to work with Software Heritage identifiers"
+description:
+  "swhid is an OCaml library to work with persistent identifiers found in Software Heritage"
+maintainer: [
+  "Léo Andrès <contact@ndrs.fr>, Dario Pinto <dario.pinto@ocamlpro.com>"
+]
+authors: [
+  "Léo Andrès <contact@ndrs.fr>, Dario Pinto <dario.pinto@ocamlpro.com>"
+]
+license: "ISC"
+homepage: "https://github.com/ocamlpro/swhid"
+bug-reports: "https://github.com/ocamlpro/swhid/issues"
+depends: [
+  "ocaml" {>= "4.0.8"}
+  "dune" {>= "2.8" & >= "2.8"}
```

Comment:
```suggestion
  "dune" {>= "2.8"}
```
This can be done by removing the `(dune (>= 2.8))` from your dune-project file.

Reply:
Hello there,

We have disabled the tests rather than vendor the files, because we do not use the files for our tests that aim merely at checking if the download link is a-ok.